app/lib/api_docs/provider_data.rb

```diff
@@ -61,12 +61,21 @@ def service_ids
       end
     end
 
+    def service_metrics
+      @account.metrics.top_level
+    end
+
+    def backend_api_metrics
+      @account.backend_api_metrics.top_level
```

Comment:
Same as https://github.com/3scale/porta/pull/1793/files#r402309363 but in this case it would be:
```suggestion
      @account.backend_api_metrics.includes(:owner).top_level
```